### DISCUSSION TITLE
Swapping Floppies

### DISCUSSION THREAD

**Discussion by CygnusTM** [erichelgeson/BlueSCSI-Toolbox#18] [Ideas]
Would it be possible to add the ability to swap floppies like you can optical disks? It would be great for doing installs.

**Comment by erichelgeson** [comment]
This repo is specifically for the Mac toolbox app - and as such Macintosh's do not support using SCSI floppies (in a meaningful way). I will be extending it to allow any non-fixed removable media such as tape, floppy, zip, MO, etc - but for Macintosh specifically (floppy) will be of no use.

**Reply by CygnusTM** [reply]
I did not know that about Macs and SCSI floppies. Thanks for the info.
